0.data_analysis/plate_eda_analyses/nbconverted/single_cell_fs_plate_cell_count_eda.py

```diff
@@ -0,0 +1,394 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Cell Count EDA of Single-Cell NF1 Plate Data
+# This analysis focuses on describing single-cell distributions within the NF1 feature-selected single-cell plate data.
```

Comment:
```suggestion
# This analysis focuses on describing single-cell count distributions within the NF1 feature-selected single-cell plate data.
```
Might also include what plates you looked at specifically instead of leaving it as just "plate data".

Reply:
Added